databaseconnect.py

```diff
@@ -169,52 +171,55 @@ def learn_question_response(H):
     cur.execute("SELECT id FROM statement_table ORDER BY id DESC")
     res = cur.fetchone()
     last_id = res[0]
-    cur.execute(f"UPDATE statement_table SET sentence='{H}' WHERE id={last_id}")    # noqa: E501
+    cur.execute("UPDATE statement_table SET sentence = '{}' WHERE id = '{}'".format(H, last_id))  # noqa: E501
     db.commit()
     B = "Thank you! I have learnt this."
     return B, 0
 
 
+@logger_config.logger
 def clear_table(table_name):
     db = connection_to_database()
     cur = db.cursor()
 
     if table_name in ("question_table", "statement_table"):
         tables_to_be_cleaned = ("question_table", "statement_table")
-        print("The following tables will be cleaned:\n")
+        cur.execute("The following tables will be cleaned:\n")
         for table in tables_to_be_cleaned:
             describe_table(cur, table)
 
         if input("Enter 'Y' to confirm cleaning of BOTH tables: ") in ("Y", "y"):   # noqa: E501
+
             for table in tables_to_be_cleaned:
                 cur.execute(f"DELETE FROM {table}")
             db.commit()
-            print("Tables cleaned successfully")
+            cur.execute("Tables cleaned successfully")
         else:
-            print("Table cleaning skipped.")
+            cur.execute("Table cleaning skipped.")
 
     else:
-        print("The following table will be cleaned:\n")
+        cur.execute("The following table will be cleaned:\n")
         describe_table(cur, table_name)
 
         if input("Enter 'Y' to confirm: ") in ("Y", "y"):
-            print("Table cleaned successfully")
-            cur.execute(f"DELETE FROM {table_name}")
+            cur.execute("Table cleaned successfully")
+            cur.execute("DELETE FROM {table_name}")
             db.commit()
         else:
-            print("Table cleaning skipped.")
+            cur.execute("Table cleaning skipped.")
```

Comment:
Can see multiple such cases. You may want to use `logging.debug` instead of `print` but ``cur.execute` won't work.

Reply:
Same as above ^